### PR TITLE
fix: revise information about absolute and fixed

### LINKED ISSUE #28209: Document content about “position” values "absolute" and "fixed"
https://developer.mozilla.org/en-US/docs/Web/CSS/position#values

we should understand "containing block " is different with "initial containing block"

I think "absolute" should be described "The element is positioned relative to its containing block", rather than "The element is positioned relative to its closest positioned ancestor (if any) or to the **initial containing block**"

especially， "fixed" should also be described like that, or **"viewport（containing block）" not "initial containing block"**，because its containing block is not "initial containing block", its containing block maybe viewport or with 'transform\filter\...'

**source document:**
"absolute
The element is removed from the normal document flow, and no space is created for the element in the page layout. The element is positioned relative to its closest positioned ancestor (if any) or to the initial [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block). Its final position is determined by the values of top, right, bottom, and left.

This value creates a new [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) when the value of z-index is not auto. The margins of absolutely positioned boxes do not [collapse](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_box_model/Mastering_margin_collapsing) with other margins.

fixed
The element is removed from the normal document flow, and no space is created for the element in the page layout. The element is positioned relative to its initial [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block). Its final position is determined by the values of top, right, bottom, and left.

This value always creates a new [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context). In printed documents, the element is placed in the same position on every page."

### PR DESCRIPTION
**Pull request by CBID2**
### Description

This PR changes the descriptions of the `absolute` and `fixed` values in the documentation about the CSS `position` property.  It also adds information about the creation of a new stacking context and the behavior of margins for these values. 

### Motivation
This PR aims to help developers gain a clearer understanding of these positioning values.

### Additional details

[This](https://developer.mozilla.org/en-US/docs/Web/CSS/position#values) is the documentation that this PR refers to. 
### Related issues and pull requests
Fixes #28209

### CODE REVIEW

File: files/en-us/web/css/position/index.md [hunk 1]
```diff
@@ -40,13 +40,13 @@ position: unset;
...
 
     This value creates a new [stacking context](/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) when the value of `z-index` is not `auto`. The margins of absolutely positioned boxes do not [collapse](/en-US/docs/Web/CSS/CSS_box_model/Mastering_margin_collapsing) with other margins.
 
 - `fixed`
 
-  - : The element is removed from the normal document flow, and no space is created for the element in the page layout. The element is positioned relative to its initial [containing block](/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block). Its final position is determined by the values of `top`, `right`, `bottom`, and `left`.
+  - : The element is removed from the normal document flow, and no space is created for the element in the page layout. The element is positioned relative to its viewport [containing block](/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block). Its final position is determined by the values of `top`, `right`, `bottom`, and `left`.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/position/index.md:49]: The sentence now sounds a bit wrong, due to the position of the link. Maybe say

> positioned relative to its [containing block](/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block), which is the viewport in the case of visual media.

?

File: files/en-us/web/css/position/index.md [hunk 2]
```diff
@@ -40,13 +40,13 @@ position: unset;
 
 - `absolute`
 
-  - : The element is removed from the normal document flow, and no space is created for the element in the page layout. The element is positioned relative to its closest positioned ancestor (if any) or to the initial [containing block](/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block). Its final position is determined by the values of `top`, `right`, `bottom`, and `left`.
+  - : The element is removed from the normal document flow, and no space is created for the element in the page layout. The element is positioned relative to its containing block [containing block](/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block). Its final position is determined by the values of `top`, `right`, `bottom`, and `left`.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/position/index.md:43]: The sentence now sounds a bit wrong, due to the position of the link, and I also think it just sounds less useful now. The bit about the closest positioned ancestor is not wrong. If anything, the linked page sounds a bit inaccurate to me. If it doesn't have a positioned ancestor, it is positioned relative to the body element, iirc.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Hi @CBID2 ! Some comments for you to consider.

File: files/en-us/web/css/position/index.md (hunk 2 above)
Comment (CBID2) [reply files/en-us/web/css/position/index.md:43]: Ok so do you suggest I remove the linked page @chrisdavidmills?

**Review by chrisdavidmills — APPROVED** [review]
Hiya @CBID2! I think what we have here now is OK. I think the `fixed` description is improved by the addition you've made, and I think the `absolute` description on this page is OK as-is. The linked containing block page might want a little bit of attention, as I described earlier, maybe at a later date.

I committed a couple of small changes directly to your PR, to fix a link that had become broken, and to remove some word duplication.

Thanks for the contribution, much appreciated.

**Comment by CBID2** [comment]
> Hiya @CBID2! I think what we have here now is OK. I think the `fixed` description is improved by the addition you've made, and I think the `absolute` description on this page is OK as-is. The linked containing block page might want a little bit of attention, as I described earlier, maybe at a later date.
> 
> 
> 
> I committed a couple of small changes directly to your PR, to fix a link that had become broken, and to remove some word duplication.
> 
> 
> 
> Thanks for the contribution, much appreciated.

Thank you for your review @chrisdavidmills

**Pull request merged**